mypy/hooks.py

```diff
@@ -0,0 +1,131 @@
+import importlib.util
+import os.path
+import pprint
+from types import ModuleType
+
+from typing import Callable, Dict, Generic, Optional, Tuple, TypeVar, TYPE_CHECKING
+
+if TYPE_CHECKING:
```

Comment:
You should use ``MYPY = False`` and then ``if MYPY: ...``. ``TYPE_CHECKING`` is not present in older versions of Python.